odl/tomo/geometry/conebeam.py

```diff
@@ -155,7 +169,8 @@ def det_refpoint(self, angle):
         circle_component = self.rotation_matrix(angle).dot(origin_to_det)
 
         # Increment by pitch
-        pitch_component = self.axis * self.pitch * angle / (np.pi * 2)
+        pitch_component = self.axis * (self.pitch_offset +
+                                       self.pitch * angle / (np.pi * 2))
```

Comment:
Honestly cant see where else, the geometries are uniquely defined by these two points, any utility  would likely just call them and get it for free?